### DISCUSSION TITLE
Vercel Subdomain not working

### DISCUSSION THREAD

**Discussion by QarthO** [vercel/community#1413] [Help] [answered]
### Summary

I have an external domain on Cloudflare. I'm attempting to set up a subdomain with this external domain so that it will redirect to my app hosted on Vercel.

ex: ``api.quartzdev.gg`` -> ``twichat-api-js.vercel.app``

On Cloudflare's end I've set up a ``CNAME`` following the documentation:
![image](https://user-images.githubusercontent.com/10179096/216038798-f241ca6e-c296-4916-b63d-1972e1684abc.png)

Then on Vercel's end I added my domain name, and it generated the SSL and looked like it was successful. I then set it to redirect to my app:

![image](https://user-images.githubusercontent.com/10179096/216039181-4357a935-2674-4c2c-af74-6e79d24341d9.png)

However, even after some time, going to [api.quartzdev.gg](https://api.quartzdev.gg/) still does not load:

![image](https://user-images.githubusercontent.com/10179096/216039291-2f51a2cb-5f87-4564-afd9-cc5a8188cdcf.png)

Did I miss a setting I need to do?

**Comment by amyegan** [comment, marked as answer]
Hi @QarthO! Can you try changing `api.quartzdev.gg` to "No Redirect" instead of redirecting it to `twichat-api-js.vercel.app`?

**Reply by QarthO** [reply]
Update: took several hours, but this change but setting it to no redirect worked! thank you!

Curious what does the difference in No Direct vs redirecting to app?

**Comment by amyegan** [comment]
I'm seeing a `404` on the home page for both URLs, but that might just be because it's supposed to be an API rather than a frontend site. I'm not sure which routes are appropriate to use for testing in this case.

If you're using something like Express.js, the guide on [using Express with Vercel](https://vercel.com/guides/using-express-with-vercel) has some helpful tips. If you're using a different framework or language, let me know and I will dig up more relevant info to help you get your project working

**Reply by QarthO** [reply]
I've already setup Express with Vercel, and the provided vercel url is working perfectly as intended for me. 
I'm just trying to get my subdomain to return the same as the provided url
